haec-user-service/src/modules/user/controllers/user.controller.ts

```diff
@@ -63,9 +63,7 @@ export class UserController {
 
   @Post('/user/verify')
   @Header('content-type', 'application/json')
-  async verifyUser(
-    @Body() verifyUserDto: VerifyUserDto,
-  ): Promise<boolean> {
+  async verifyUser(@Body() verifyUserDto: VerifyUserDto): Promise<boolean> {
```

Comment:
I think `this and the following changes are just syntax`, I kept getting `End of Line errors` and I wasn't able to resolve them.